src/ui/reducers/app.ts

```diff
@@ -85,7 +85,7 @@ export default function update(
     }
 
     case "set_analysis_points": {
```

Comment:
do you think it's time to add an analysis reducer?

Reply:
Let's do that when it gets more complex.